pkg/csi/service/node.go

```diff
@@ -1176,9 +1182,7 @@ func publishFileVol(
 	if params.ro {
 		mntFlags = append(mntFlags, "ro")
 	}
-	if cnstypes.CnsClusterFlavor(os.Getenv(csitypes.EnvClusterFlavor)) == cnstypes.CnsClusterFlavorGuest {
-		mntFlags = append(mntFlags, "hard")
-	}
+	mntFlags = append(mntFlags, "hard")
```

Comment:
why are we using 'hard' option here? if we have to use it, we should couple it with 'intr' option as a best practice.